src/pkg/zoci/pull.go

```diff
@@ -105,12 +97,11 @@ func (r *Remote) LayersFromRequestedComponents(ctx context.Context, requestedCom
 			return nil, fmt.Errorf("component %s does not exist in this package", name)
 		}
 	}
+	var images []string
 	for _, component := range pkg.Components {
 		// If we requested this component, or it is required, we need to pull its images and tarball
 		if slices.Contains(requestedComponents, component.Name) || component.Required {
-			for _, image := range component.Images {
-				images[image] = true
-			}
+			images = append(images, component.Images...)
```

Comment:
Wouldn't doing this cause multiple of the same image to show in the list?

Reply:
Yes my mistake, good catch